crates/fap-lld.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# Helper script for linking and post-processing FAP binaries.
+
+import os
+import sys
+from subprocess import run
+
+TOOLS_PATH = '../tools'
+
+
+def main():
+    args = sys.argv[1:]
+    print(args)
+
+    # Run the linker with the given arguments.
+    result = run(
+        [
+            'cargo',
+            'run',
+            '--quiet',
+            '--release',
+            '--bin',
+            'fap-lld',
+            '--',
+        ] + args,
+        cwd=os.path.join(os.path.dirname(__file__), TOOLS_PATH),
+    )
+    if result.returncode:
+        sys.exit(result.returncode)
```

Comment:
For a completed process, there should always be a valid `returncode`, so it's OK to call this unconditionally.